Makefile

```diff
@@ -5,8 +5,10 @@ ASM ?= nasm
 PEG ?= meta/peg.py
 ENC ?= meta/enc.py
 
+INSTALL_DIR ?= /opt/naive/
+
 COMMON_CFLAGS := $(CFLAGS) -std=c99 -Werror -Wall -Wextra -Wstrict-prototypes \
-	-Wformat
+	-Wformat -Wimplicit-fallthrough -Wno-strict-prototypes
```

Comment:
`-Wno-strict-prototypes` can be removed now that I've fixed the cases this was firing.